contents/docs/hogql/index.mdx

```diff
@@ -26,6 +26,8 @@ SQL insights enables you to create and edit insights using full SQL queries inst
 
```

Comment:
Annoying, I cant comment on lines you haven't changed:

On line 11, can you please also add a hyperlink to clickhouse sql i.e.

"HogQL is effectively a wrapper around [ClickHouse SQL](https://clickhouse.com/docs/en/sql-reference)"